README.md

```diff
@@ -1,3 +1,7 @@
 # wcc
 
 Low level language that compiles directly to WebAssembly. The goal of the project is to be as close to WebAssembly as possible yet providing nice abstraction mechanisms to speed up the development process. The target audience of the language is people who do [raw WebAssembly](https://surma.dev/things/raw-wasm/) development for recreational purposes.
+
+# Hacktoberfest
+
+Give me a t-shirt pls thx
```

Comment:
@aodhneine I think since this is an official documentation, you should not abbreviate both `pls` and `thx`. Consider changing that.
```suggestion
Give me a t-shirt, please. Thank you!
```

Reply:
BOOMER OMEGALUL

Reply:
Looks like my linter fails on these changes. I am afraid there must be an error or something.
```
Warning: unknown directive #Hacktoberfest
in a.cpp  at line 1 col 1
a.cpp:3:18: Error: Parse error
Found:
  > Give me a t-shirt, please. Thank you!


1 hint
```

Reply:
Which linter are you using? I'd like to see if I can reproduce the issue.

Reply:
`hlint v2.1.26`

Reply:
@Nhlest same here:

```
[nico@sagittarius ~]$ hlint --version
HLint v2.1.26, (C) Neil Mitchell 2006-2019
```

Reply:
It looks like `hlint` is failing in a different place for me. Sadly I'm unable to reproduce your issue, @Nhlest.

```
Warning: unknown directive #wcc
in README.md  at line 1 col 1
Warning: unknown directive #Hacktoberfest
in README.md  at line 5 col 1
README.md:3:68-69: Error: Parse error: on input `of'
Found:
  > Low level language that compiles directly to WebAssembly. The goal of the project is to be as close to WebAssembly as possible yet providing nice abstraction mechanisms to speed up the development process. The target audience of the language is people who do [raw WebAssembly](https://surma.dev/things/raw-wasm/) development for recreational purposes.


1 hint
```

Reply:
Have you tried updating `hlint` to the latest version? I'm using `HLint v3.1.6`.

Reply:
Due to the fact that my issue could not be reproduced on the latest version of hlint, i retract my complaint. Looks good to me :+1: Commit approved.

Reply:
:+1: